keyboards/system76/launch_lite_1/keymaps/default/keymap.c

```diff
@@ -77,5 +78,22 @@ ________________________________________________________________________________
 };
 
 bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+    case KC_LCTL:
+        lctl_pressed = record->event.pressed;
+        break;
+    case KC_RCTL:
+        rctl_pressed = record->event.pressed;
+        break;
+    case KC_ESC:
+        esc_pressed = record ->event.pressed;
+        break;
+    };
     return true;
 }
+
+void matrix_scan_user(void) {
+    if (lctl_pressed && rctl_pressed && esc_pressed) {
+        reset_keyboard();
```

Comment:
Looks like `system76_ec_unlock()` is used in `launch_1/launch_1.c`.

Reply:
Looks like that works! Thanks!